fe/fe-core/src/main/java/com/starrocks/load/ExportChecker.java

```diff
@@ -147,4 +153,34 @@ private void runExportingJobs() {
             }
         }
     }
+
+    private boolean checkBeStatus(ExportJob job) {
+
+        boolean beHasErr = false;
+        String errMsg = "";
+        for (Long beId : job.getBeStartTimeMap().keySet()) {
+            Backend be = GlobalStateMgr.getCurrentSystemInfo().getBackend(beId);
```

Comment:
Will there be any periodic reports between the BE and the FE about an export job? I think checking for the execution status of BE about the export job will cover more cases, for example when BE is alive but get stuck.

Reply:
Yes u are right，this PR just handle a case that be restarted. I think if we want to handle all bad caes, like backend stuck, backend be dropped, and so on, we need to modify export's scheduling framework. This may be a big change, not quite sure if it needs to be done now. For now, I want to fix this bad case firstly